.env

```
@@ -0,0 +1,79 @@
+# AstraGuard AI Environment Configuration
+# This file contains local configuration. DO NOT commit this file to version control.
+
+# ============================================================================
+# LOGGING & DEBUG
+# ============================================================================
+LOG_LEVEL=INFO
+DEBUG=false
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DEBUG key should go before the LOG_LEVEL key

(UnorderedKey)